### PR TITLE
mention use of ls -i in tutorial

### LINKED ISSUE #394: mention in tutorial about ls -i meaning
```
$ ls -i data/Posts.xml.zip
78483929 data/Posts.xml.zip

$ ls -i .dvc/cache/ec/
78483929 88519f8465218abb23ce0e0e8b1384
```

In [tutorial page](https://dvc.org/doc/tutorial/define-ml-pipeline), where we mention this code block, we can tell that it prints the index no of each file, and inode in cache and data/ is same. 

Should I mention @shcheklein this as a comment in code block, or in documentation?

### PR DESCRIPTION
**Pull request by kurianbenoy**
Fix #394

### CODE REVIEW

File: static/docs/tutorial/define-ml-pipeline.md [hunk 1]
```diff
@@ -118,6 +118,8 @@ $ ls -i .dvc/cache/ec/
 $ du -sh .
  41M .
 ```
+Do note that `ls -i` prints prints the index number of each file and index
```
Comment (shcheklein) [inline static/docs/tutorial/define-ml-pipeline.md]: Remove `Do`, no need to emphasize this

Comment (shcheklein) [reply static/docs/tutorial/define-ml-pipeline.md]: number (`123344678`) specify the number to make it easier to understand

Comment (shcheklein) [reply static/docs/tutorial/define-ml-pipeline.md]: `prints prints` -> `prints`

File: static/docs/tutorial/define-ml-pipeline.md [hunk 2]
```diff
@@ -118,6 +118,8 @@ $ ls -i .dvc/cache/ec/
 $ du -sh .
  41M .
 ```
+Do note that `ls -i` prints prints the index number of each file and index 
+number for `data/Posts.xml.zip` and `.dvc/cache/ec` remains same always.
```
Comment (shcheklein) [inline static/docs/tutorial/define-ml-pipeline.md]: `.dvc/cache/ec` is a directory, it contains a file `88519f8465218abb23ce0e0e8b1384` that has the same inode as `data/Posts.xml.zip`

Comment (shcheklein) [reply static/docs/tutorial/define-ml-pipeline.md]: `remains same always` -> `remained same`

**Comment by ryokugyu** [comment]
@shcheklein are these changes done, or I resolve them? 
@kurianbenoy you working on it?

**Comment by shcheklein** [comment]
@kurianbenoy have you forgotten to push a fix for the resolved conversations?

**Comment by kurianbenoy** [comment]
@shcheklein I forgot to push the changes made. Thanks for reminding @ryokugyu

**Comment by shcheklein** [comment]
@kurianbenoy np! will you push them though? :)

**Comment by kurianbenoy** [comment]
> @kurianbenoy np! will you push them though? :)

Now I have pushed @shcheklein . Just tell me if anymore changes needs to be done

**Comment by shcheklein** [comment]
thanks, @kurianbenoy !

**Pull request merged**